gapir/cc/surface.cpp

```diff
@@ -168,13 +168,15 @@ void handleWindow(uint32_t width, uint32_t height) {
   }
 }
 
+static const int32_t stream_index = 0;
```

Comment:
can't see where this is used, remove?

Reply:
If I remember correctly this was one of the suspects in a problem that I had while trying to compile. I will retry and see if it's actually needed or not. Because it's static I can't be 100% sure.

Reply:
Done!